mtest/CMakeLists.txt

```diff
@@ -40,6 +40,8 @@ if (MSVC)
 endif (MSVC)
 
 QT5_ADD_RESOURCES(qrc_files ${PROJECT_SOURCE_DIR}/mtest/mtest.qrc
+            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Leland.qrc
+            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Edwin.qrc
```

Comment:
For some reason `${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Campania.qrc` is missing in this file

Reply:
Thanks, solved ;-)